hardware/Comply-to-the-Regulations.md

```diff
@@ -0,0 +1,394 @@
+
+  - Spurs emission limits
+    General requirement is very loose in this part, the limit is -20dBc everywhere. But for the spurs lie in restrict bands defined in Part-15.205, the spurs including harmonics will be less than -49dBm for frequency < 960MHz, and be less than -41dBm for frequency >960MHz. For the harmonics limits, the strength limit can be relax by using DCF factor. Because radiated spurious emissions above 1 GHz are AVERAGE measurements covered by FCC Part 15.35.
+    Basically it needs higher order(5-order) low pass filter since the fundamental power is very high(+30dBm).
+
+  - Rx requirements
+    
+    No sensitivity requirement. But the limits for harmonics or spurs in a restricted band is defined in “FCC Part 15.209 Unintentional Radiators” as following:
+    - Spurious falls below 960 MHz = -49.2 dBm EIRP (max) (15.209.a);
+    - Spurious falls above 960 MHz = -41.2 dBm EIRP (max) (15.209.a).
+
+## 4.2. FCC requirements in 2.4GHz band
+For 2.4GHz band regulation, it is defined in the FCC-Part-15.247 and FCC-Part-15.249. FCC-Part-15.247 defined high power application, and FCC-Part-15.249 defined low power application. 
+- Part-15.247
+  Typical applications are Weather station reporting, Automatic meter reading, and long range communication in this regulation. It covers wide bandwidth applications in 2.4GHz band:
+  a)Frequency Hopping Spread Spectrum (FHSS);
+  b)Wideband digital modulation (BWMOD_6dB > 500 kHz);
+  c)Direct Sequence Spread Spectrum; 
+  - Tx power limits
+    Since it is a wide band appliction in this regulation. The maximum output power can be high since the band "spread out" effection. The fundamental output power should be a Peak measurement.
+     - Wideband digital modulation → 1 watt (+30 dBm) max;
+     - FHSS (> 75 hopping channels) → 1 watt (+30 dBm) max;
+     - FHSS (> 15 hopping channels) → 0.125 watt (+21 dBm) max.
+
+  - Tx bandwidth requirement
+     For FHSS operation system, the maximum allowed 20dB bandwidth of hopping channel is 500KHz. But for using digital modulation techniques, the minimum 6dB bandwidth shall be at least 500KHz.
+   
+  - Power spectral density(PSD) limit
+    This requirement is for digitally modulated system. The PSD should not be greater than 8dBm in any 3KHz band during any time interval of continuous transmission.
+
+  - Spurs emission limits
+    General requirement is very loose in this part, the limit is -20dBc everywhere. But for the spurs lie in restrict bands defined in Part-15.205, the spurs including harmonics will be less than -49dBm for frequency < 960MHz, and be less than -41dBm for frequency >960MHz. 2nd, 3rd, 5th, 6th, 8th, 9th and 10th harmonics of 2.4 GHz is in the restrict bands. For the harmonics limits, the strength limit can be relax by using DCF factor.Because radiated spurious emissions above 1 GHz are AVERAGE measurements covered by FCC Part 15.35.
```

Comment:
Change "lie in restrict bands" to "lying in restricted bands".  Add space to change ">960MHz" to "> 960MHz".  Change "relax by using DCF factor.Because" to "relaxed by using DCCF because"

Reply:
Corrected.